views/admin/connect.php

```diff
@@ -27,12 +27,11 @@
 
         \core\Settings::add('access_token',$new_access_token);
         \core\Settings::add('refresh_token',$new_refresh_token);
-        \core\Settings::add('expiry',$new_expires_in);
+        \core\Settings::add('expiry',$new_expires_in + time());
+        $appSettings = \core\Settings::get();

```

Comment:
Is this basically the main change?

Reply:
Yep, my bad I was gonna comment that this was the main change.